.circleci/config.yml

```
@@ -74,9 +74,11 @@ jobs:
       # see https://circleci.com/orbs/registry/orb/adobe/helix-post-deploy
       # for more available parameters
       - helix-post-deploy/monitoring:
+          targets: aws
           statuspage_name: Helix Content Bus
           statuspage_group: Publishing
           newrelic_group_policy: Publishing Repeated Failure
+          newrelic_group_targets: aws
```

```suggestion
          # newrelic_group_targets: aws
```
In case of a single `target`, or if only the first one should be added to the group policy, this parameter can be omitted.